tests/data/test_contrastive_dataloader.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2024 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import torch
+
+from llmfoundry.utils.builders import build_dataloader
+from tests.data_utils import (
+    temporary_contrastive_streaming_dataset,
+    temporary_tokenizer,
+)
+
+
+@pytest.mark.parametrize('ds_format', ['text_a_text_b', 'query_passages'])
```

Comment:
Nit: can you change the parametrization since you updated the enum name to something like `one_query_one_response` and `one_query_multiple_responses`